Kickstarter-iOS/Views/Controllers/SignupViewController.swift

```diff
@@ -44,8 +63,11 @@ internal final class SignupViewController: UIViewController, MFMailComposeViewCo
     _ = self.disclaimerButton
       |> disclaimerButtonStyle
 
-    _ = self.emailTextField
-      |> emailFieldStyle
+    _ = self.nameTextField
+      |> UITextField.lens.returnKeyType .~ .next
```

Comment:
there is a style for `nameTextField` on line 75, can you add this lens to that line to condense them?